src/main/resources/jenkins/branch/Messages.properties

```diff
@@ -28,6 +28,9 @@ DescriptionColumn.displayName=Project description
 ItemColumn.DisplayName=Name
 NamedExceptionsBranchPropertyStrategy.DisplayName=Named branches get different properties
 NoTriggerBranchProperty.suppress_automatic_scm_triggering=Suppress automatic SCM triggering
+NoTriggerBranchProperty.strategy.all=Suppress all, runs will not be trigger by indexing or webhook events
```

Comment:
```suggestion
NoTriggerBranchProperty.strategy.all=Suppress all, runs will not be triggered by indexing or webhook events
```

Reply:
I applied the suggestion and squashed the commits 🙂